packages/hashes.csv

```diff
@@ -0,0 +1,28 @@
+fetchai/connections/gym,Qmd8Z9wKPTBxXXGaB6cZRPn1UhNvDQ4T8T72LiiMNR2omW

```

Comment:
Note, IPFS hashes make the packages content addressable. This hash can be used to upload the package to IPFS. Then the package is uniquely addressable with the hash. So at some point in the future one can retrieve the package via IPFS directly rather than via the registry. The registry could run an IPFS node.